### DISCUSSION TITLE
Why does `brew update` not show the list of updated packages?

### DISCUSSION THREAD

**Discussion by HYAdev** [Homebrew/discussions#4293] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.4
ORIGIN: https://github.com/Homebrew/brew
HEAD: 4d0154fa7926e8a7b06e9b074dd45957e820407c
Last commit: 3 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 8b559cd3c47e6a88d8999b2b6d0e5713697e80fa
Core tap last commit: 13 days ago
Core tap branch: master
Core tap JSON: 02 Mar 23:53 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.86.0 => /usr/bin/curl
macOS: 13.2.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Usually, when I `brew update`, it shows a list of the updated formulas/casks, however recently, it hasn't shown anything after updating, it just shows `Already up-to-date.` Addtionally, when I run `brew upgrade` it proceeds to upgrade outdated packages. I do not believe I modified homebrew in any way.

**Comment by paguilera** [comment]
I was coming to see if someone else had already reported this issue as I'm seeing this as well while using an M2 mac. 

Here's the results of my `brew config` after having just completed running `brew upgrade`.

```
HOMEBREW_VERSION: 4.0.4
ORIGIN: https://github.com/Homebrew/brew
HEAD: 4d0154fa7926e8a7b06e9b074dd45957e820407c
Last commit: 4 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: db25ff891d07186085be417e472cdde13881d2c7
Core tap last commit: 2 weeks ago
Core tap branch: master
Core tap JSON: 03 Mar 14:27 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.39.2 => /opt/homebrew/bin/git
Curl: 7.85.0 => /usr/bin/curl
macOS: 13.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
Rosetta 2: false
```

**Comment by beaudh** [comment]
There appears to be some confusion with developers as to the nature of this as it is apparently reported as fixed, but then a decision was made to depreciate the the functionality due to limitations with the github API that is now used for sourcing formulas and casks. If you wish to go down the rabbit hole:

https://github.com/Homebrew/brew/issues/14702
https://github.com/Homebrew/brew/issues/14706
https://github.com/Homebrew/brew/issues/14720
https://github.com/Homebrew/brew/pull/14802
https://github.com/Homebrew/brew/pull/14866

**Comment by juanleung** [comment]
To fix this issue I created an alias and chained two commands
`brew update && brew outdated`
